x-pack/plugins/observability/server/routes/rules.ts

```diff
@@ -14,10 +14,27 @@ const alertsDynamicIndexPatternRoute = createObservabilityServerRoute({
   options: {
     tags: [],
   },
-  handler: async ({ ruleDataClient }) => {
-    const reader = ruleDataClient.getReader({ namespace: observabilityFeatureId });
+  params: t.type({
+    query: t.type({
+      registrationContexts: t.array(t.string),
+      namespace: t.string,
```

Comment:
I heard that it's always `default` in Observability. I guess it's not correct?

Reply:
It is at the moment, but I'd rather have the alerts page decide about the hard-coded value TBH.

Reply:
Makes sense to me 👍